netlib/multidict.py

```diff
@@ -171,6 +171,14 @@ def items(self, multi=False):
         else:
             return super(_MultiDict, self).items()
 
+    def clear(self, key):
+        """
```

Comment:
Let me ponder that. I've always found the .pop interface messy and am in the habit of adding a .clear to my implementations of the interface.